pkgs/by-name/ds/dsnet/package.nix

```
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  buildGoModule,
+}:
+
+buildGoModule (finalAttrs: {
+  pname = "dsnet";
+  version = "0.8.1";
+
+  src = fetchFromGitHub {
+    owner = "naggie";
+    repo = finalAttrs.pname;
```

The name of the repository isn't necessary the same as the package name. In fact, the package would be broken when someone overrides the package name. Not ideal

```suggestion
    repo = "dsnet";
```

---

Good point